sdk/identity/test-resources.bicep

```diff
@@ -9,12 +9,16 @@ param baseName string = resourceGroup().name
 @description('The location of the resource. By default, this is the same as the resource group.')
 param location string = resourceGroup().location
 
-param runtime string = 'node'
-var functionWorkerRuntime = runtime
+param sshPubKey string
+
+param adminUserName string = 'azureuser'
 
 //See https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles
 var blobContributor = subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'ba92f5b4-2d11-453d-a403-e96b0029c9fe') //Storage Blob Data Contributor
 var websiteContributor = subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'de139f84-1756-47ae-9be6-808fbbe84772') //Website Contributor
+// cluster parameters
+var kubernetesVersion = '1.26.6'
```

Comment:
@christothes You may want to parse `az aks get-versions -o json` from the pre script to get the latest version and add it to template parameters. The minor versions move pretty quickly (every couple months or so) and AKS only supports the latest 3 minor versions.